test/DurableTask.SqlServer.Tests/Utils/TestService.cs

```diff
@@ -205,6 +206,14 @@ public void RegisterInlineActivity(string name, string version, TaskActivity act
             this.worker.AddTaskActivities(new TestObjectCreator<TaskActivity>(name, version, activity));
         }
 
+        public void RegisterInlineOrchestration<TOutput, TInput>(
+            string orchestrationName,
+            Func<OrchestrationContext, TInput, Task<TOutput>> implementation,
+            Action<OrchestrationContext, string, string> onEvent = null)
```

Comment:
This is a new convenience overload that doesn't require the caller to pass the `version` parameter (which we rarely use).